infra/modules/service/access_logs.tf

```diff
@@ -0,0 +1,95 @@
+locals {
+  purpose = "access-logs"
+}
+
+resource "aws_s3_bucket" "alb" {
+  bucket_prefix = "${var.service_name}-${local.purpose}"
+  force_destroy = true
+  # checkov:skip=CKV2_AWS_62:Ensure S3 buckets should have event notifications enabled
+  # checkov:skip=CKV_AWS_18:Ensure the S3 bucket has access logging enabled
+  # checkov:skip=CKV_AWS_144:Ensure that S3 bucket has cross-region replication enabled  
+  # checkov:skip=CKV_AWS_300:Ensure S3 lifecycle configuration sets period for aborting failed uploads
+  # Known issue where Checkov gets confused by multiple rules
+}
+
+resource "aws_s3_bucket_public_access_block" "block" {
```

Comment:
I added it because it was not complicated and Chekhov wanted it. I believe the difference is that one can potentially allow public access via ACL or bucket policy. The block means that even if those are in place, it still won't be made publicly available

Reply:
got it! i do see that we did this for other buckets too.

in that case can we rename the resource to something more specific than "block"? let's be consistent with our places we are using aws_s3_bucket_public_access_block, so in this case it should be called `"aws_s3_bucket_public_access_block" "access_logs"`